### DISCUSSION TITLE
Infobox like in Wikipedia/Mediawiki

### DISCUSSION THREAD

**Discussion by astrdev** [squidfunk/mkdocs-material#5757] [Q&A] [answered]
Hello everyone. I wanted to know, is it possible to create infoboxes like the one available in Wikipedia/Mediawiki (here is an example below) on the right of the page? 
Thanks in advance for your reply and have a nice day.

![Screenshot_20230725_103129](https://github.com/squidfunk/mkdocs-material/assets/82945541/22eb274e-902e-48ce-a843-482282796e83)

**Comment by squidfunk** [comment, marked as answer]
You can achieve something that is pretty close by using [inline admonitions](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#inline-blocks).

``` markdown
!!! info inline end "Lorem ipsum"

    Lorem ipsum dolor sit amet, consectetur
    adipiscing elit. Nulla et euismod nulla.
    Curabitur feugiat, tortor non consequat
    finibus, justo purus auctor massa, nec
    semper lorem quam in massa.
```

Other than that, with some extra CSS, you can also restyle admonitions to look more like the image you posted.

**Reply by astrdev** [reply]
Oh nice I'd never heard of it. Thank you for this information 😄
